src/alire/alire.ads

```diff
@@ -102,32 +102,35 @@ package Alire with Preelaborate is
           'a' .. 'z' | 'A' .. 'Z' | '0' .. '9' | '_' | Extension_Separator);
    --  Used for cross-platform folder names
 
-   subtype Platform_Independent_Path is String;
-   --  This type is used to ensure that folder separators are externally always
-   --  '/', and internally properly converted to the platform one
+   subtype Base_Path is String;
```

Comment:
Things are much better with your introduced distinctions, thanks! Still, I find `Base_Path` a bit confusing because it evokes `basename` to me, which is entirely a different thing. What do you think about using `Any_Path` instead?

Reply:
I'm ok with `Any_Path`.